src/Facade/ResourceBuilders/PurchaseOrderReqStateResourceBuilder.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Linn.Purchasing.Facade.ResourceBuilders
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Linn.Common.Facade;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+    using Linn.Purchasing.Resources;
+
+    public class PurchaseOrderReqStateResourceBuilder : IBuilder<PurchaseOrderReqState>
+    {
+        public PurchaseOrderReqStateResource Build(PurchaseOrderReqState entity, IEnumerable<string> claims)
+        {
+            if (entity == null)
```

Comment:
possibly don't need this check?

Reply:
Yeah think you're right, I do need this sort of check in the actual Req resource builder for getting the application state I'm sure, but I don't need it here for ReqStates